roles/kubernetes-apps/persistent_volumes/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Can you make this role a subdirectory of persistent_volumes? Let's mimic the behavior of network plugins and use meta/main.yml to include subroles if cloud_provider is openstack. Then you can get rid of all the when cases.